server/src/models/people/populate.js

```diff
@@ -1,46 +1,87 @@
 import _ from "lodash";
 
-import { people_years } from "../constants.js";
 import { get_standard_csv_file_rows } from "../load_utils.js";
 
-import {
-  camel_case_headcount_model_names,
-  snake_case_headcount_model_names,
-} from "./headcount_model_utils.js";
-
-const headcount_csv_names_by_headcount_model_name = _.chain(
-  camel_case_headcount_model_names
-)
-  .zip(snake_case_headcount_model_names)
-  .fromPairs()
-  .value();
-
-const format_csv_records_then_register_to_headcount_models = (
-  headcount_models_and_records
-) => {
-  _.each(headcount_models_and_records, (model_and_record) => {
-    _.each(model_and_record.records, (record) =>
-      _.each(
-        people_years,
-        (col) => (record[col] = record[col] && parseFloat(record[col]))
-      )
-    );
-    _.each(model_and_record.records, (record) =>
-      model_and_record.model.register(record)
-    );
-  });
-};
-
-export default function ({ models }) {
-  const headcount_models_and_records = _.map(
-    headcount_csv_names_by_headcount_model_name,
-    (headcount_csv_name, headcount_model_name) => ({
-      model: models[headcount_model_name],
-      records: get_standard_csv_file_rows(`${headcount_csv_name}.csv`),
-    })
-  );
-
-  format_csv_records_then_register_to_headcount_models(
-    headcount_models_and_records
-  );
+export default async function ({ models }) {
+  const {
+    EmployeeAgeGroup,
+    EmployeeAvgAge,
+    EmployeeExLvl,
+    EmployeeFirstOfficialLang,
+    EmployeeGender,
+    EmployeeRegion,
+    EmployeeType,
+  } = models;
+
+  let employee_age_rows,
+    employee_avg_age_rows,
+    employee_ex_lvl_rows,
+    employee_fol_rows,
+    employee_gender_rows,
+    employee_region_rows,
+    employee_type_rows;
```

Comment:
Hmm, `let`s are usually a "code smell" to me since they normally indicate mutation. In this case they don't though, right? It's because the variables want to be used in the scope above the `try` block where they're actually declared.

I suggest below to not use the try/catch anyway, but in general there's a few ways you could get around this, e.g. put the try/catch in a function that returns the row variables as keys in an object, and then use a destructuring declaration like `const {a, b, c} = function()` to declare the variables back in to the scope you want them in.

Of course, that's not necessary without the try/catch.